automatic/Ghostscript.app/Ghostscript.app.nuspec

```diff
@@ -1,16 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>Ghostscript (Install)</title>
+    <id>Ghostscript.app</id>
+    <version>9.21</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/Ghostscript.app</packageSourceUrl>
     <owners>chocolatey</owners>
+    <title>Ghostscript (Install)</title>
     <authors>Artifex</authors>
-    <licenseUrl>http://www.gnu.org/licenses/agpl-3.0.html</licenseUrl>
     <projectUrl>http://ghostscript.com/</projectUrl>
     <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/ghostscript.svg</iconUrl>
+    <copyright>Artifex Software, Inc</copyright>
+    <licenseUrl>http://www.gnu.org/licenses/agpl-3.0.html</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>http://git.ghostscript.com/?p=ghostpdl.git;a=summary</projectSourceUrl>
+    <docsUrl>http://www.ghostscript.com/doc/9.21/Readme.htm</docsUrl>
+    <bugTrackerUrl>https://bugs.ghostscript.com/</bugTrackerUrl>
+    <tags>ghostscript.app ghostscript postscript admin foss cross-platform</tags>
+    <summary>an interpreter for the PostScript language and for PDF.</summary>
     <description>
 Ghostscript is a package of software that provides:
 
```

Comment:
good point, I didn't actually look too closely on the existing description.
I'll have a look at it and see if it's suited to be in a features list.